linker/Tests/Tests/ParseResponseFileLinesTests.cs

```diff
@@ -0,0 +1,63 @@
+using NUnit.Framework;
+using System.Collections.Generic;
+
+namespace Mono.Linker.Tests {
+	[TestFixture]
+	public class ParseResponseFileLinesTests {
```

Comment:
Can you add tests that have Windows and Linux style file paths in the response file contents?  Historically file paths are one of the most common source of command line parsing woes in my experience so having coverage of file paths is reassuring.

Reply:
Fixed